tests/installation/ipxe_install.pm

```diff
@@ -70,13 +70,18 @@ sub set_bootscript {
     my $arch = get_required_var('ARCH');
     my $autoyast = get_var('AUTOYAST', '');
     my $mirror_http = get_required_var('MIRROR_HTTP');
+    my $openqa_url = get_var('OPENQA_URL', 'openqa.suse.de');
+    my $agama_iso = get_var('ISO');
 
     # trim all strings from variables to get rid of bogus whitespaces
     $arch =~ s/^\s+|\s+$//g;
     $autoyast =~ s/^\s+|\s+$//g;
     $mirror_http =~ s/^\s+|\s+$//g;
+    $openqa_url =~ s/^\s+|\s+$//g;
+    $agama_iso =~ s/^\s+|\s+$//g;
 
-    my $install = "install=$mirror_http";
+    my $agama_installer = get_var('AGAMA_INSTALLER', "http://$openqa_url/assets/iso/$agama_iso");
```

Comment:
not sure if this is a new variable, but it would be better if you use a name for narrow to what you are doing and why you need it (ipxe).

Reply:
Thanks, you are correct, it is not defined in https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/variables.md as well. let me fix it later

Reply:
I would prefer that would not be defined to be honest, in Yam squad we don't use settings as much as possible so we would prefer not be ping about this in the future, if this is a variable you use for ipxe setup should reflect in the name that is for that purpose or that is related with an iso path (I don't really know what it means).
Could you use a simple variable to store the value? or you really need to pass it? if that is so, please redefine its name so it will point to some specific scope.